lib/svg_sprite/svg_sprite.rb

```diff
@@ -189,8 +189,28 @@ module SvgSprite
 
   FA_ICON_MAP = { 'far fa-' => 'far-', 'fab fa-' => 'fab-', 'fas fa-' => '', 'fa-' => '' }
 
-  SVG_SPRITE_PATHS = Dir.glob(["#{Rails.root}/vendor/assets/svg-icons/**/*.svg",
-                               "#{Rails.root}/plugins/*/svg-icons/*.svg"])
+  CORE_SVG_SPRITES = Dir.glob("#{Rails.root}/vendor/assets/svg-icons/**/*.svg")
+
+  THEME_SPRITE_VAR_NAME = "icons-sprite"
+
+  def self.custom_svg_sprites(theme_ids = [])
+    custom_sprite_paths = Dir.glob("#{Rails.root}/plugins/*/svg-icons/*.svg")
+
+    ThemeField.where(name: THEME_SPRITE_VAR_NAME, theme_id: Theme.transform_ids(theme_ids))
```

Comment:
Can we be slightly more specific here - just in case someone decides to name a SCSS field "icons-sprite". I think this should do it:
```suggestion
    ThemeField.where(type_id: ThemeField.types[:theme_upload_var], name: THEME_SPRITE_VAR_NAME, theme_id: Theme.transform_ids(theme_ids))
```